website/content/en/status/report-2024-01-2024-03/clusteradm.adoc

```diff
@@ -0,0 +1,37 @@
+=== Cluster Administration Team
+
+Links: +
+link:https://www.freebsd.org/administration/#t-clusteradm[Cluster Administration Team members] URL: link:https://www.freebsd.org/administration/#t-clusteradm[]
+
+Contact: Cluster Administration Team <clusteradm@FreeBSD.org>
+
+FreeBSD Cluster Administration Team members are responsible for managing the machines the Project relies on to synchronize its distributed work and communications.
+
+In this quarter, the team has worked on the following:
+
+* Regular support for FreeBSD.org user accounts.
+* Regular disk and parts support (and replacement) for all physical hosts and mirrors.
+* Set up a new mirror in Chicago.
+More details are provided https://freebsdfoundation.org/blog/powering-up-the-future-the-new-freebsd-cluster-in-chicago/[here].
+
+==== FreeBSD Official Mirrors Overview
+
+Current locations are Australia, Brazil, Germany, Japan (two full mirror sites), Malaysia, South Africa, Sweden, Taiwan, United Kingdom (full mirror site), United States of America -- California, Chicago, New Jersey (primary site), and Washington.
+
+The hardware and network connection have been generously provided by:
+
+* https://www.bytemark.co.uk/[Bytemark Hosting] (being decommissioned)
+* Cloud and SDN Laboratory at https://www.bbtower.co.jp/en/corporate/[BroadBand Tower, Inc]
+* https://www.cs.nycu.edu.tw/[Department of Computer Science, National Yang Ming Chiao Tung University]
+* https://deploy.equinix.com/[Equinix]
+* https://internet.asn.au/[Internet Association of Australia]
+* https://www.isc.org/[Internet Systems Consortium]
+* https://www.inx.net.za/[INX-ZA]
+* https://www.kddi-webcommunications.co.jp/english/[KDDI Web Communications Inc]
+* https://www.mohe.gov.my/en/services/research/myren[Malaysian Research & Education Network]
+* https://www.metapeer.com/[MetaPeer]
+* https://www.nyi.net/[New York Internet]
+* https://nic.br/[NIC.br]
+* https://www.teleservice.net/[Teleservice Skåne AB] (new since 2023Q4)
+* https://your.org/[Your.Org]
+
```

Comment:
How about adding something like "Please contact us if you want to provide an official mirror."

Reply:
done